### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by steven-murray**
Adds some simple documentation, requested by the JOSS reviewers.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -8,13 +8,9 @@ little bit helps, and credit will always be given.
...
-features. You can do the following:
-
-    * `Report a bug <https://github.com/rasg-affiliates/21cmSense/issues/new?template=bug_report.md>`
-    * `Request a Feature <https://github.com/rasg-affiliates/21cmSense/issues/new?template=feature_request.md>`
-    * `Ask a Question <https://github.com/rasg-affiliates/21cmSense/issues/new?template=question.md>`
-
-When doing any of these, please try to be as succinct, but detailed, as possible, and use
+features. You can do any one of these, or simply ask a question about how to use 21cmSense,
+by filing an issue `here <https://github.com/rasg-affiliates/21cmSense/issues/new>`_.
+When doing tjos, please try to be as succinct, but detailed, as possible, and use
```
Comment (mkolopanis) [inline CONTRIBUTING.rst:13]:
```suggestion
When doing this, please try to be as succinct, but detailed, as possible, and use
```

**Review by mkolopanis — COMMENTED** [review]
one quick typo fix

File: CONTRIBUTORS.rst [hunk 2]
```diff
@@ -1,11 +1,8 @@
...
-======
-
-`Jonathan Pober <https://github.com/jpober`_: jonathan_pober@brown.edu
-
-Contributors
-------------
+Authors
+=======
 
+`Jonathan Pober <https://github.com/jpober>`_
```
Comment (mkolopanis) [inline CONTRIBUTORS.rst:4]: Is jonnie meant to not be bulleted?

**Review by mkolopanis — COMMENTED** [review]
one quick question, don't know why I didn't see it before.

File: CONTRIBUTORS.rst (hunk 2 above)
Comment (steven-murray) [reply CONTRIBUTORS.rst]: yes, yes he is.

Comment (steven-murray) [reply CONTRIBUTORS.rst]: fixed it up

**Review by mkolopanis — APPROVED** [review]

**Pull request merged**